app/src/utils/subscan.ts

```diff
@@ -1,9 +1,9 @@
 import { subscan, environment } from '@snowbridge/api'
 import { TransferStatus } from '@snowbridge/api/dist/history'
-import { SubscanTransferResponse, FromParachainTrackingResult } from '@/models/subscan'
+import { SubscanTransferResponse, FromParachainTrackingResult, Status } from '@/models/subscan'
 import { OngoingTransferWithDirection } from '@/models/transfer'
 
-export const trackFromParachainTx = async (
+export const trackWithinPolkadotTx = async (
```

Comment:
We could shorten it and make it more specific:

```suggestion
export const trackXcm = async (
```